MAUI/Maps/getting-started.md

```diff
@@ -10,22 +10,41 @@ keywords: .net maui maps, .net maui maps library.
 
 # Getting started with MAUI Maps (SfMaps)
 
-This section explains the steps required to add the maps control with the shape layer and its elements such as data labels, tooltip, markers, and legends. This section covers only basic features needed to know to get started with Syncfusion maps.
+This section explains the steps required to add the maps control with the shape layer and its elements such as data labels, tooltip, markers, and legends. This section covers only basic features needed to know to get started with Syncfusion maps. Follow the steps below to add a basic maps view to your project.
 
 To get start quickly with our .NET MAUI Maps, you can check the below video.
 
 {% youtube
 "youtube:https://www.youtube.com/watch?v=ibUB70vM5DU"%}
 
-## Creating an application using the .NET MAUI Maps
+## Prerequisites
 
-* Create a new .NET MAUI application in the Visual Studio.
+Before proceeding, ensure the following are setup:
+1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later is installed.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here.](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code)
 
-* Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add SfMaps to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Maps] then install that.
+## Step 1: Create a New MAUI Project
```

Comment:
<img width="299" alt="image" src="https://github.com/user-attachments/assets/73c702fb-4360-4532-a462-76231ffa3469">

add the step 1 properly

Reply:
Updated the changes